docker/Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN npm i ./puppeteer-core-latest.tgz ./puppeteer-latest.tgz \
     && groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
     && mkdir -p /home/pptruser/Downloads \
     && chown -R pptruser:pptruser /home/pptruser \
+    && mkdir /home/.npm \
```

Comment:
why is the cache in /home/.npm and not in /hom/pptruser/.npm? The documentation seems to suggest that it should be in /home/pptruser

Reply:
I am not sure why it is not in `/home/pptruser`. 

Try running `RUN npm config get cache` in this docker return `/home/.npm`. Another alternative of the above `mkdir. chown` is to add a line to set cache location `npm config set cache "/home/pptruser/.npm"`.